Modules/main_module.rb

```diff
@@ -0,0 +1,39 @@
+module MainModule
+  def add_game(app)
+    puts "\n"
+    puts 'Is game multiplayes [Yes/No]:'
+    multiplayer = gets.chomp
+    puts 'When is game last played [Enter date format YYYY-MM-DD]:'
+    last_played_at = gets.chomp
+    puts 'When is game published [Enter date format YYYY-MM-DD]:'
+    publish_date = gets.chomp
+    app.create_game(multiplayer, last_played_at, publish_date)
+  end
+
+  def add_mucicalbum(app)
+    puts "\n"
+    puts 'Is it on spotify [True/False]:'
+    on_spotify = gets.chomp
+    if %w[True true].include?(on_spotify)
+      on_spotify = true
+    elsif %w[False false].include?(on_spotify)
+      on_spotify = false
+    else
+      puts 'Invalid input'
+    end
+    puts 'When is album published [Enter date format YYYY-MM-DD]:'
+    publish_date = gets.chomp
+    app.create_musicalbum(publish_date, on_spotify)
+  end
+
+  def add_book(app)
+    puts "\n"
+    puts 'Who is the publisher [Enter Publisher Name]:'
+    publisher = gets.chomp
+    puts 'What Is the cover\'s state like? [Enter good or bad]:'
+    cover_state = gets.chomp
+    puts 'When was the book published [Enter date format YYYY-MM-DD]:'
+    publish_date = gets.chomp
+    app.create_book(publisher, cover_state, publish_date)
+  end
```

Comment:
[REQUIRED] I like the fact that you found a workaround for the scenario where a user enters an invalid date in most places in your app but you missed the point where the user tries to add a book. Have a look at the screenshot below:
<img width="1440" alt="image" src="https://github.com/Dorcas126/Ruby-Group-capstone/assets/62938735/ff1a2236-7f66-4049-9a0b-a58ffe2dde4b">

Please prevent this error from happening